iphone/Classes/TiUtils.m

```diff
@@ -201,6 +205,15 @@ +(BOOL)isIOS9_1OrGreater
     return [UITouch instancesRespondToSelector:@selector(altitudeAngle)];
 }
 
++(BOOL)isIOS9_3OrGreater
+{
+#if IS_XCODE_7_3
+    return [WCSession instancesRespondToSelector:@selector(activationState)];
+#else
+    return NO;
+#endif
```

Comment:
Isn't there another new method you can use to verify if it's iOS9.3 or greater? then we don't have to import watchconnectivity framework.